Sources/Request/Request/RequestParams/RequestConfiguration.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//  
+//
+//  Created by brennobemoura on 15/11/20.
+//
+
+import Foundation
+
+protocol SessionConfiguration: RequestParam {
```

Comment:
I think this should be called `SessionParam` to match `RequestParam`. Also please make it `public` so users can make custom ones.